Update api.md (#2196)
infiniflow/ragflow: docs/references/api.md

    "retmsg": "success"
}
```
### Demo for Upload File(Python) 

```python
# upload_to_kb.py  
import requests

def upload_file_to_kb(file_path, kb_name, token='ragflow-xxxxxxxxxxxxx', parser_id='naive'):
    """  

>     Uploads a file to a knowledge base via a specified API.  

Reviewer: ```suggestion
    Uploads a file to a knowledge base.  
```

    Args:  

>     - file_path: Path to the file to be uploaded.  

Reviewer: ```suggestion
    - file_path: Path to the file to upload.  
```

    - kb_name: Name of the target knowledge base.  

>     - parser_id: Parser ID to use for file parsing (default: 'naive').  

Reviewer: ```suggestion
    - parser_id: ID of the chosen file parser (defaults to 'naive').  
```
